.babelrc

```diff
@@ -1,3 +1,7 @@
 {
-  "presets": ["es2015", "react"]
+  "sourceType": "unambiguous",
```

Comment:
This was needed because the default in babel 7 is "modules" which was causing issues with unknown tokens. For more details: https://babeljs.io/docs/en/options#sourcetype